config/v3_0_experimental/types/disk.go

```diff
@@ -91,6 +94,20 @@ func (n Disk) partitionNumbersCollide() bool {
 	return false
 }
 
+func (d Disk) partitionLabelsCollide() bool {
```

Comment:
How about also returning the dupe label so we print it as part of the error?

Reply:
This gets at a more fundamental problem in that pretty much all of the validation code doesn't provide adequate context. It's something I've wanted to fix but haven't have the time. If it's not a huge issue to you right now I'd prefer to fix it all at once.

Reply:
Are you ok punting this down the road?

Reply:
Yup, that's fine!